package/gluon-status-page/src/js/lib/helper.js

```diff
@@ -106,7 +106,9 @@ define([ "bacon" ], function (Bacon) {
         "not connected": "nicht verbunden",
         "Packets/s": "Pakete/s",
         "Statistic": "Statistik",
-        "Neighbors": "Nachbarknoten"
```

Comment:
Why changing this to "Nachbarn"?

Reply:
I don't really see the point either, good translations are always about context.  "Nachbarknoten" sounds good, it doesn't need to match English version exactly.

Reply:
:)